Modelltisztázások/modell_5v2/modell_5v2.mod

```diff
@@ -156,13 +156,17 @@ for{b in Buszok}
 {
   printf "Busz %d:\n",b;
   printf "Ossz futott km  / Osszfogyasztas / maxtoltes: %g / %g / %g\n",osszhasznalat[b],osszfogyasztas[b],maxtoltes[b];
+  printf "\tJaratok/Toltesek:\n";
   for{j in MindenJarat:hozzarendel[j,b]=1}
-    printf "\tJarat %s: %s(%g) --%g--> %s(%g) (%g)->(%g)\n",j,honnan_minden[j],mikortol_minden[j],tav2_minden[j],hova_minden[j],meddig_minden[j],toltottsege[j,b],toltottsegu[j,b];
+    printf "\t\tJarat %s: %s(%g) --%g--> %s(%g) (%g)->(%g)\n",j,honnan_minden[j],mikortol_minden[j],tav2_minden[j],hova_minden[j],meddig_minden[j],toltottsege[j,b],toltottsegu[j,b];
+  
+  printf "\tAtmenetek:\n";
   for{j in MindenJarat: elsojarat[j,b]=1}
-    printf "\tElsojarat: Depo --%g--> Jarat %s : %s(%g) -> %s(%g) (%g)->(%g)\n",tav[depo[b],honnan_minden[j]],j,depo[b],0,honnan_minden[j],mikortol_minden[j],toltottsege[j,b],toltottsegu[j,b];
+    printf "\tElsojarat: Depo --%g--> Jarat %s : %s(%g) -> %s(%g) (%g)->(%g)\n",tav[depo[b],honnan_minden[j]],j,depo[b],0,honnan_minden[j],mikortol_minden[j],toltottsege[j,b]+tav[depo[b],honnan_minden[j]]*fogyasztas[b],toltottsege[j,b];
   for{j in MindenJarat, j2 in MindenJarat: atmenet[b,j,j2]=1}
-    printf "\tAtmenes: Jarat %s --%g--> Jarat %s : %s(%g) -> %s(%g) (%g)->(%g)\n",j,tav[hova_minden[j],honnan_minden[j2]],j2,hova_minden[j],meddig_minden[j],honnan_minden[j2],mikortol_minden[j2],toltottsege[j,b],toltottsegu[j,b];
+    printf "\tAtmenes: Jarat %s --%g--> Jarat %s : %s(%g) -> %s(%g) (%g)->(%g)\n",j,tav[hova_minden[j],honnan_minden[j2]],j2,hova_minden[j],meddig_minden[j],honnan_minden[j2],mikortol_minden[j2],toltottsegu[j,b],toltottsege[j2,b];
```

Comment:
itt is a j jarat elotti, utani toltottsegek voltak kiirva, nem pedig az, hogy j-rol j2-be atmenetkor mennyirol mennyire megy le.